main/tests/server/src/com/google/refine/tests/model/recon/StandardReconConfigTests.java

```diff
@@ -172,4 +173,17 @@ public void deserializeColumnDetail() throws JsonParseException, JsonMappingExce
     	ColumnDetail cd = ParsingUtilities.mapper.readValue(uiJson, ColumnDetail.class);
     	TestUtils.isSerializedTo(cd, backendJson);
     }
+    
+    @Test
+    public void deserializeReconResult() throws JsonParseException, JsonMappingException, IOException {
+    	String json = "{\"score\":100.0,"
+    			+ "\"match\":false,"
+    			+ "\"type\":["
```

Comment:
should be "types"?

Reply:
No, the JSON format implement with `org.json` used `type`, not `types`. We need to keep this format so that reconciliation services still work.